src/index.d.ts

```diff
@@ -0,0 +1,298 @@
+// Type definitions for Paylike
+
+import * as Promise from 'bluebird'
+
+declare class Paylike {
+  constructor(appKey: string)
+
+  transactions: Paylike.Transactions
+  merchants: Paylike.Merchants
+  cards: Paylike.Cards
+  apps: Paylike.Apps
+  service: Paylike.Service
+}
+
+declare namespace Paylike {
+
+  export interface PaylikeError extends Error {
+    message: string
+  }
+  export interface PaylikeErrorConstructor extends Error {
+    new (message: string): PaylikeError
+    (message: string): PaylikeError
```

Comment:
I would assume so. I'm not 100% sure, this is from https://github.com/Microsoft/TypeScript/blob/master/lib/lib.es6.d.ts